core/components/ErrorMessage.js

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import { Helmet } from 'react-helmet';
+import Icon from './Icon';
+
+const ErrorMessage = () => (
+  <Fragment>
+    <Helmet>
+      <style>{'body{margin:0}'}</style>
```

Comment:
Just curious, why did you use `<Helmet>` instead of [createGlobalStyle ](https://www.styled-components.com/docs/api#createglobalstyle)?

Reply:
Yup, no reason to use `helmet` being able to use `createGlobalStyle`.

Reply:
Would you mind switching it to `createGlobalStyle` to maintain coherence with the rest of our codebase?

You can also take the opportunity to do a `fix()` commit at the same time and solve the `refactor()` issue.

Reply:
Right, that's what I'm going to do.

Reply:
Thanks @DAreRodz! 😁